for-import/temp-buttons.css

```diff
@@ -0,0 +1,93 @@
+@import"../for-import/reset.css";
+@import"../for-import/colors.css";
+
+.general-style-section {
+    margin-bottom: 2rem;
+}
+
+.btn {
+    padding: 10px;
+    width: 6rem; 
+    /* margin-right: 10px; */
+    font-weight: bold;
+    border-radius: 5px;
+    border: 0;
+    cursor: pointer
+
```

Comment:
empty line can be removed wherever not required

Reply:
removed all